### PR TITLE
[State API] Add Node ID + Job ID + Actor ID to list tasks and list actors

### PR DESCRIPTION
**Pull request by rkooo567**
## Why are these changes needed?

Just did it for fun..

This PR adds Node ID + Job ID + Actor ID to list tasks and list actors.

## Checks

- [ ] I've signed off every commit(by using the -s flag, i.e., `git commit -s`) in this PR.
- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

**Comment by rkooo567** [comment]
@rickyyx ideally we also want to add duration (task) + uptime (actor) by 2.2...

File: python/ray/experimental/state/common.py [hunk 1]
```diff
@@ -464,6 +477,15 @@ class TaskState(StateSchema):
     #: breakdowns and typical state transition flow.
     #:
     scheduling_state: TypeTaskStatus = state_column(filterable=True)
+    #: The job id of this actor.
```
Comment (rickyyx) [inline python/ray/experimental/state/common.py:480]:
```suggestion
    #: The job id of this task.
```

File: python/ray/experimental/state/common.py [hunk 2]
```diff
@@ -464,6 +477,15 @@ class TaskState(StateSchema):
...
+    job_id: str = state_column(filterable=True)
+    #: The node id of the task. If the task is retried, it could
```
Comment (rickyyx) [inline python/ray/experimental/state/common.py:482]:
```suggestion
    #: Id of the node that runs the task. If the task is retried, it could
```

**Review by rickyyx — APPROVED** [review]
<div><br/>via <a href="https://giphy.com/netflix/">NETFLIX</a> on <a href="https://giphy.com/gifs/netflix-killer-queen-the-babysitter-kDZMoj30w3OswLR08v">GIPHY</a></div>


Just comments. Couldn't wait to use this.

**Review by rickyyx — APPROVED** [review]

**Review by scv119 — APPROVED** [review]

**Pull request merged**